### ISSUE TITLE
Page Authorize Its not showing in local development

### ISSUE THREAD

**Issue by ariansyahAkbarAtech** [polkadot-js/extension#1238]
* **I'm submitting a ...**

  - [✔ ] Bug report
  - [ ] Feature request
  - [ ] Support request
  - [ ] Other

* **What is the current behavior and expected behavior?**
I test clone this repo & run local on chrome browser Version 111.0.5563.66 (Official Build) (64-bit) but Authorize Page Its Not showing we must hit Connected Button

* **What is the motivation for changing the behavior?**
 i test download from official polkadotjs extension and connect to wallet Authorize Page Its ShowUp in extension before connect to account
 
![image](https://user-images.githubusercontent.com/100116567/227138573-e3df4c3c-7ee9-4bc1-9b6b-65dea433f795.png)

* **Please tell us about your environment:**

  - Version:
  - Environment:
  - windows 10
  - chrome browser Version 111.0.5563.66 (Official Build) (64-bit)

    - [ ] Node.js
    - [✔ ] Browser
    - [ ] Other (limited support for other environments)

  - Language:

    - [ ] JavaScript
    - [✔ ] TypeScript (include tsc --version)
    - [ ] Other

**Comment by jacogr** [comment]
The master version has changed quite a bit with regards to authorization - instead of passing through all acocunts, you need to select specific accounts.

TL;DR This is entirely expected, as per the master version.

**Comment by TarikGul** [comment]
Closing as this is answered
